### DISCUSSION TITLE
Mi band 5 slow update

### DISCUSSION THREAD

**Discussion by axelct** [ESPresense/ESPresense#331] [Q&A]
Hi all,
I'm experiencing some delay problem with my Mi band 5. When I enter in a room there about 2 minutes to see my smartband in the room.
Could be an update interval of Mi band or can I tweak some configuration on espresense to avoid this delay?
Thanks in advance.
Ax

**Comment by DTTerastar** [comment]
Do you know how often the mi band advertises?

**Reply by axelct** [reply]
unfortunally not :-(
I think this is the problem, but I haven't found any information about that and seems it cannot be changed at all
